README.md

```diff
@@ -18,98 +18,62 @@ those dependencies are provided by [npm][npmjs].
 Hubot has a HTTP listener which listens on the port specified by the `PORT`
 environment variable.
 
-You can specify routes to listen on in your scripts by using the `router`
-property on `robot`.
+For the lulu implementation, it is recommended to run it with `--disable-httpd`
+since irc.case.edu is a public-facing server.
 
-```coffeescript
-module.exports = (robot) ->
-  robot.router.get "/hubot/version", (req, res) ->
-    res.end robot.version
-```
+##  Persistence
 
-There are functions for GET, POST, PUT and DELETE, which all take a route and
-callback function that accepts a request and a response.
+The `hubot-redis-brain` package is currently activated for lulu. This requires
+an implementation of Redis in some way or another and the appropriate configuration. For further information, see [hubot-retis-brain doc](https://www.npmjs.com/package/hubot-redis-brain).
 
```

Comment:
Notes of interest to include on persistence with Redis:

- I discovered that the REDIS_URL environment variable needs to look like this: `redis://:password@address.com:port/prefix`.  Note the colon before the password.  I think it's a bug in the implementation, but I'm not sure.  Regardless, it doesn't work without the colon.
- You don't need the REDIS_URL environment variable if you're hosting redis on the same server as Lulu.  Plus, then you can bind Redis to localhost only, and disable authentication (as long as it's a private server).
- Maybe mention that I currently maintain redis on ACM-People?  Or maybe that info will just go out of date way too quickly.  Just a thought.